From 70f79b9b93c02a7c1444ae42734e8d32dda03a4f Mon Sep 17 00:00:00 2001
From: Pawan Gupta <pawan.kumarx.gupta@intel.com>
Date: Wed, 15 Jun 2016 12:21:44 -0700
Subject: [PATCH 423/441] firmware: renesas/K2018090.mem for only Puma6

renesas/K2018090.mem is only needed for Puma6. Guarded the
compilation with CONFIG_X86_PUMA6 flag.

Signed-off-by: Pawan Gupta <pawan.kumarx.gupta@intel.com>
---
 firmware/Makefile | 2 ++
 1 file changed, 2 insertions(+)

--- a/firmware/Makefile
+++ b/firmware/Makefile
@@ -135,7 +135,9 @@ fw-shipped-$(CONFIG_USB_SERIAL_XIRCOM) +
 fw-shipped-$(CONFIG_USB_VICAM) += vicam/firmware.fw
 fw-shipped-$(CONFIG_VIDEO_CPIA2) += cpia2/stv0672_vp4.bin
 fw-shipped-$(CONFIG_YAM) += yam/1200.bin yam/9600.bin
+ifeq ($(CONFIG_X86_PUMA6),y)
 fw-shipped-$(CONFIG_USB_XHCI_HCD) += renesas/K2018090.mem
+endif
 fw-shipped-all := $(fw-shipped-y) $(fw-shipped-m) $(fw-shipped-)
 
 # Directories which we _might_ need to create, so we have a rule for them.
